### DISCUSSION TITLE
data_graph

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/labview_book#54] [Announcements]
# data_graph

能够方便的把数据以图形化的方式展现出来，是 LabVIEW 的一大特色。LabVIEW 为此提供了丰富的控件和函数，可以大大简化图形化显示数据的复杂度。LabVIEW 图形控件选板上列出了几十种控件，分别用于显示不同类型的数据。但实际上，真正用于显示的部分，不需要分成那么多种控件，比如，一个图片控件既可以显示显示用直角坐标系表示数据，也可以显示用极坐标系表示的数据。所以，读者们可能已经发现了，这个选板上的很多控件被拖到前面板上时，它还会在 VI 的程序框图上添加一些代码。这些代码是真正用来处理不同类型的数据的，它们把不同类型的数据转换为图形控件可以显示的方式，再使用图形控件把数据绘制出来。

https://lv.qizhen.xyz/data_graph

**Comment by foraiurs** [comment]
想问一下堆叠模式（Stack Plots）或者 “覆盖模式（Overlay Plots）的选项具体在哪里？我在右键菜单没找到

**Reply by ruanqizhen** [reply]
先确认一下是不是 波形图表控件 (chart) ，只有 波形图表控件 有堆叠模式，波形图控件 (graph) 没有堆叠模式。

**Reply by zhangning1639** [reply]
中文名称改了，叫层叠显示曲线和分格显示曲线。（2020版）

**Comment by alvin-lrl** [comment]
https://lv.qizhen.xyz/data_graph#%E6%97%B6%E9%A2%91%E8%81%94%E5%90%88%E8%B0%B1%E5%9B%BE
图中的magnitude是什么单元啊，找了半天还找不到

**Reply by ruanqizhen** [reply]
你是问图标上有个眼镜和一个音符的那个 VI 吗？
这是 LabVIEW 读取音频文件的 VI，在函数选板  Programming -> Graphics & Sound -> Sound -> Files 下面

**Reply by alvin-lrl** [reply]
不是，是右边那个，写着magnitude的。
另外，这个写着power的控件，在2022 Q3的版本里面是不是已经被取代？我也找不到，只找到了一个类似的‘FFT Power Spectrum and PSD.vi’

**Reply by ruanqizhen** [reply]
哦，这个是簇的“解除绑定”，用于把簇里的元素取出来，参考： https://lv.qizhen.xyz/data_array#%E7%B0%87%E6%95%B0%E6%8D%AE%E8%BF%90%E7%AE%97

**Reply by alvin-lrl** [reply]
天哪，它只显示了一行的时候就认不出来了 T＾T

**Reply by ruanqizhen** [reply]
power的控件确实不见了，‘FFT Power Spectrum and PSD.vi’ 跟它的接线端一模一样，看似可以替代。我还没有测试过。正几天测试一下，再把这个示例更新一下。

**Reply by alvin-lrl** [reply]
Ｔｈｘ！

**Comment by XuY6969** [comment]
请问一下，在三维曲面中 Surfuce Plot helper控件在2018中没找到

**Reply by XuY6969** [reply]
找到了，打扰您了
